### PR TITLE
Restructure cli commands

### PR DESCRIPTION
**Pull request by xoxys**
Fixes: https://github.com/woodpecker-ci/woodpecker/issues/3866

Personally, I would keep the `pipeline` command at the top-level but open for other suggestions. What do you think about sorting commands alphabetically? To me, the current command order looks a bit random.

### CODE REVIEW

File: cli/repo/deploy/deploy.go
[the remark this answers is not in the document]
Comment (xoxys) [reply cli/repo/deploy/deploy.go:1]: Good point. I just used the layout from the issue and didn't think too much about it 🙈

**Comment by xoxys** [comment]
> Could you add a migration instruction / hint to the docs?

Sure but what do you have in mind? The new structure is covered by the docs and the change itself is listed as breaking change in the release notes already.

**Comment by anbraten** [comment]
I would suggest adding something like to https://woodpecker-ci.org/migrations#user-migrations

```
Some cli commands got restructured to provide a clear structure. Use:
- `cli secret --global ...` is now `cli admin secret ...`
- `cli secret --org ...` is now `cli org secret ...`
- ...
```

My goal before releasing 3.0 would be to have a migration list a user / admin could easily follow like a manual.

**Comment by xoxys** [comment]
OK. IMO that's a bit overkill and frankly very tedious, but I can look into it.

**Pull request merged**